compass/ocean/tests/isomip_plus/initial_state.py

```diff
@@ -234,10 +233,12 @@ def run(self):
                                sectionY=section_y, dsMesh=ds, ds=ds,
                                showProgress=show_progress)
 
-        ds['landIcePressure'] = ds['landIcePressure'].expand_dims(dim='Time', axis=0)
+        ds['landIcePressure'] = \
+            ds['landIcePressure'].expand_dims(dim='Time', axis=0)
         ds['bottomDepth'] = ds['bottomDepth'].expand_dims(dim='Time', axis=0)
         ds['totalColThickness'] = ds['ssh']
-        ds['totalColThickness'].values = numpy.sum(ds['layerThickness'].values, axis=2)
+        ds['totalColThickness'].values = \
+            ds['layerThickness'].sum(dim='nVertLevels')
```

Comment:
The `numpy.sum()` function doesn't handle NaNs well, whereas the xarray sum does (and it has the added advantage of summing over a named dimension rather than a numbered axis).  My runs of the "generic" Ocean0 test case were showing masked-out column thickness wherever there were NaNs in `layerThickness` (when bathymetry was above the top of the bottom layer).